src/services/lnd/schema.types.d.ts

```diff
@@ -0,0 +1,16 @@
+interface LnPaymentType {
+  id: string
+  _id: string
```

Comment:
same as above? maybe no need to expose both?

Reply:
I wasn't sure if mongo would need both... was following example from other schema types we have